drake/CMakeLists.txt

```diff
@@ -162,11 +162,14 @@ file(WRITE ${CMAKE_INSTALL_PREFIX}/matlab/addpath_drake.m
 	   "  wd = cd(fullfile(mfiledir,'${relpath}'));\n"
 	   "  addpath_drake();\n"
 	   "  cd(wd);\n"
+     "end\n"
+     "\n"
     )
 
 file(WRITE ${CMAKE_INSTALL_PREFIX}/matlab/rmpath_drake.m
 	   "function rmpath_drake()\n"
 	   "  mfiledir = fileparts(which(mfilename));\n"
+     "  jarfiledir = fullfile(mfiledir, '..', 'share', 'java');\n"
```

Comment:
i don't think this accomplishes anything?  and it doesn't look like you actually remove the jarfile (rmpath_drake isn't updated?)